client/packages/requisitions/src/RequestRequisition/DetailView/columns.tsx

```diff
@@ -216,9 +227,11 @@ export const useRequestColumns = () => {
       warningForExcessRequest,
       showExtraColumns,
       usesRemoteAuthorisation,
+      showIndicativeUnitPriceInRequisitions,
       maxMonthsOfStock,
       plugins.requestRequisitionLine?.tableColumn,
       errors,
+      t,
```

Comment:
@CarlosNZ told me we're not adding this in  as a dependancy on our new tables as it should available. Not sure the downside though. @CarlosNZ care to comment?

Reply:
IDK if other don't have linters running or ignore them but throughout our code base basic react lint rules complain about missing dependencies in memoization/callbacks not to mention useEffects. I went down the rabbit hole to check if t is actual generally a stable reference but it's a bit of typescript hell. Wish instead the type system actually made it clear, that'd be so sweet.